modules/telco-core-agent-based-installer.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/telco_core_ref_design_specs/telco-core-rds.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="telco-core-agent-based-installer_{context}"]
+= Agent-based Installer
+
+New in this release::
+* No reference design updates in this release
+
+Description::
+Telco core clusters can be installed using the Agent-based Installer.
+This method allows you to install OpenShift on bare-metal servers without requiring additional servers or VMs for managing the installation.
+The Agent-based Installer can be run on any system (for example, from a laptop) to generate an ISO installation image.
+The ISO is used as the installation media for the cluster supervisor nodes.
+Progress can be monitored using the ABI tool from any system with network connectivity to the supervisor node's API interfaces.
+
+ABI supports the following:
+
+* Installation from declarative CRs
+* Installation in disconnected environments
+* No additional servers required to support installation, for example, the bastion node is not longer needed
```

Comment:
Can you drop this last bullet from the list and still be accurate? The bullet should really take a verb (_**are**_ required). But that will make it a sentence, which requires you to make the other two bullets into sentences where it is not needed for comprehension. 

```suggestion
ABI supports the following:

* Installation from declarative CRs
* Installation in disconnected environments

No additional servers are required to support installation, for example, the bastion node is no longer needed.
```

If not, watch the typo on _**not** longer needed_.